### PR TITLE
Update to 732

### PR DESCRIPTION
**Pull request by AterAnimAvis**
- [x] Update Names
- [x] Update Roles
- [X] Update PFP
- [x] Mark NSFW
- [x] Emoji

### CODE REVIEW

File: quotes/data.json [hunk 1]
```diff
@@ -3352,6 +3383,221 @@
...
+            "user": "ravenbuilder934",
+            "text": "I'm [...] a [...] old [...] tall [...] lady"
+        },
+        {
+            "user": "ravenbuilder934",
+            "text": "I need [...] to beat shirmp"
+        },
+        {
+            "user": "FinnT730",
+            "text": "these eggs [...] are [...] horny people sometimes...."
```
Comment (ravenbuilder934) [inline quotes/data.json:3414]: nsfw?

File: quotes/data.json [hunk 2]
```diff
@@ -3352,6 +3383,221 @@
...
+            "user": "SizableShrimp",
+            "text": "mines better it has gore"
+        },
+        {
+            "user": "EERussianguy",
+            "text": "sorry as an end user it's required that I can't read"
+        },
+        {
+            "user": "AshersLab",
+            "text": "f*** off, it's long"
```
Comment (ravenbuilder934) [inline quotes/data.json:3446]: nsfw?

File: quotes/data.json [hunk 3]
```diff
@@ -3352,6 +3383,221 @@
...
+            "side_text": "quoting himself",
+            "text": "the clothes don't make the man lmao"
+        },
+        {
+            "user": "SizableShrimp",
+            "text": "I can do it if Curle wants me to"
+        },
+        {
+            "user": "ravenbuilder934",
+            "text": "oh yeah one sec I'll just wipe my ass with some coke"
```
Comment (ravenbuilder934) [inline quotes/data.json:3492]: nsfw

File: quotes/data.json [hunk 4]
```diff
@@ -3352,6 +3383,221 @@
...
+            "user": "AshersLab",
+            "text": "but let's not look like a bunch of assholes xD"
+        },
+        {
+            "user": "SizableShrimp",
+            "text": "yes I am stealing a child named jackson"
+        },
+        {
+            "user": "Silk",
+            "text": "Are there sex toys?"
```
Comment (ravenbuilder934) [inline quotes/data.json:3521]: nsfw

File: quotes/data.json [hunk 5]
```diff
@@ -3352,6 +3383,221 @@
...
+            "user": "jriwanek",
+            "text": "#He'sNotTheMessiahHe'sAVeryNaughtyBoy"
+        },
+        {
+            "user": "SizableShrimp",
+            "text": "OwO"
+        },
+        {
+            "user": "AshersLab",
+            "text": "your face sucks"
```
Comment (ravenbuilder934) [inline quotes/data.json:3533]: nsfw?

File: quotes/data.json [hunk 6]
```diff
@@ -3352,6 +3383,221 @@
...
+        },
+        {
+            "user": "jriwanek",
+            "text": "Beats a penis flytrap"
```
Comment (ravenbuilder934) [inline quotes/data.json:3537]: nsfw

File: quotes/data.json [hunk 7]
```diff
@@ -3352,6 +3383,221 @@
...
+            "side_text": "to <strong style=\"font-size: 115%\">Raven</strong>",
+            "text": "you'll never be atop me"
+        },
+        {
+            "user": "Amethyst",
+            "text": "i'm the quote dominatrix of this server"
+        },
+        {
+            "user": "jriwanek",
+            "text": "Look, I'm watching smurff porn, OKAY?"
```
Comment (ravenbuilder934) [inline quotes/data.json:3558]: nsfw

File: quotes/data.json [hunk 8]
```diff
@@ -3352,6 +3383,221 @@
...
+        },
+        {
+            "user": "OakBricks",
+            "text": "how rip off tits"
```
Comment (ravenbuilder934) [inline quotes/data.json:3562]: nsfw

File: quotes/data.json [hunk 9]
```diff
@@ -3352,6 +3383,221 @@
...
+        },
+        {
+            "user": "ravenbuilder934",
+            "text": "random take your coherency out of here"
+        },
+        {
+            "user": "ravenbuilder934",
+            "text": "do i <i>fuck</i> the IRS? no not yet"
```
Comment (ravenbuilder934) [inline quotes/data.json:3570]: nsfw

File: quotes/data.json [hunk 10]
```diff
@@ -3352,6 +3383,221 @@
...
+            "user": "AshersLab",
+            "text": "RIP CurlieWurlie 2021, [...] Slapped to death"
+        },
+        {
+            "user": "ravenbuilder934",
+            "text": "what does thos have to do wirh ni thpigh"
+        },
+        {
+            "user": "Maci",
+            "text": "....you want me to our condoms on my arrows?"
```
Comment (ravenbuilder934) [inline quotes/data.json:3582]: nsfw

File: quotes/data.json [hunk 11]
```diff
@@ -3352,6 +3383,221 @@
...
+        },
+        {
+            "user": "SizableShrimp",
+            "side_text": "aka <strong style=\"font-size: 115%\">SizableShirmp</strong>",
+            "text": "something something twerking"
+        },
+        {
+            "user": "SizableShrimp",
+            "side_text": "aka <strong style=\"font-size: 115%\">SizableShirmp</strong>",
+            "text": "or maybe it requires the cock nibbler image"
```
Comment (ravenbuilder934) [inline quotes/data.json:3592]: nsfw

**Comment by AterAnimAvis** [comment]
Update to 732, all tasks completed

**Review by sciwhiz12 — APPROVED** [review]
:shipit:

**Pull request merged**